modules/ecs-base/main.tf

```
@@ -24,3 +24,8 @@ resource "aws_iam_policy" "otel" {
     ]
   })
 }
+
+resource "aws_service_discovery_http_namespace" "internal_namespace" {
+  name        = var.service_discovery_namespace_name
```

Lets give this name based on the existing props in the stack

```suggestion
  name        = "{var.namespace}.{var.stage}.service-connect"
```